common/test/common/AmpValidityTest.scala

```diff
@@ -24,14 +33,21 @@ trait AmpValidityTest extends FlatSpec with Matchers with ConfiguredTestSuite {
       val io = new ProcessIO(commandInputWriter, BasicIO.toStdOut, BasicIO.toStdErr)
 
       // Pass the content to the command line tool (external process) via stdin ('-' option)
-      val process = "node_modules/amphtml-validator/index.sh -".run(io)
+      val process = s"node_modules/.bin/amphtml-validator --validator_js ${fetchValidator()} -".run(io)
 
       withClue("AMP validator should complete with exit value 0, the actual exit value of ") {
         process.exitValue() should be(0)
       }
     }
   }
 
+  private def fetchValidator(): String = {
+    recorder.load(VALIDATOR_URI) {
+      wsClient.url(VALIDATOR_URI).get()
+    }
+    recorder.fileLocation(VALIDATOR_URI)
```

Comment:
I am not sure I understand why `fileLocation` is useful here. Does the `load` not returning the filepath (wrapped in a future)?